### ISSUE TITLE
using nvda unable to read windows 11 updates status when browsing objects

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13719]
### Steps to reproduce:
I hope I can show my anguish.
Enter windows update from windows 11 and start a large update preferably.
Now navigate with tabe and as we know the focus is pushed to the pause updates button but this call is not about this problem.
So I tried using object navigation, found a list, browsed through the list but nvda didn't read the update percentage.
I turned on narrator and used screen exploration with incert + space, just navigated with arrows like this and narrator perfectly read the update name and update percentage.
what an easy and effective uncomplicated experience the narrator offers in this case, wow, we desperately need to give the nvda user a similar experience, I was really impressed.
### Actual behavior:
using object navigation nvda does not announce the status of updates, percentage of download and installation progress.
### Expected behavior:
with my lowest expectations when using object navigation it should be able to read update install status, percentage of install.
In the ideal world I believe we should provide a screen exploration experience as easy as screen exploration in the narrator activated with incert + space.
But I believe that this will not be addressed in this call, the writings are left to register my surprise at how well the narrator worked in this case and that nvda should provide the user with a similar experience.
So the real request of this call is to be able to read the install percentage of updates using object navigation.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-25456,6cf16d1a

#### Windows version:
11 10.0.22000.675
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:

**Comment by josephsl** [comment]
Hi, actually, you can do so – this is because the update percentage is the “grandchild” of the update entry. The update list item contains (has a child) a grouping, which in turn contains two children: the update label and the download/install percentage. The reason why Narrator can show you this info when you press up arrow is because the scan mode can move through elements regardless of hierarchy, which NVDA can “emulate” if you assign keyboard commands to move between objects through a “flow model” just like you can do wit touch interaction (move between objects regardless of level). Thanks.

**Comment by fernando-jose-silva** [comment]
thanks for the feedback, this configuration to make the navigation by objects go through all the objects without caring about the hierarchy, and something that I as a regular user, with no programming knowledge, can do?
 If so can you teach me?  sometimes I found the child within the update, but many times, I get to the list, entered the list, I find update, but when I go to check if the children, the update has no children.

**Comment by josephsl** [comment]
Hi, two ways: using ObjPad add-on (a version compatible with NVDA 2022.1 is on its way), or assigning keyboard shortcuts from input gestures dialog. There is a third way, but it might be confusing at first: provided people know where to look, add a setting in review cursor settings panel to toggle what we are talking about (good for new dev please). Thanks.

**Comment by fernando-jose-silva** [comment]
I can't test on updates specifically, as there are no updates at the moment, but it assigns commands for flat navigation and reading the information on the screen about windows worked correctly, it was possible to read the information without the complexity of entering or leaving items, very good for beginner users, or even for us when we want ease, or we get lost in the hierarchies that seem infinite.
I agree with
@josephsl 
Please provide the user with a quick and easy way to use the flat navigation, this will help many users, beginners or lost like me.

**Comment by cary-rowen** [comment]
Hi, @josephsl 
Are you receiving translations via SVN before OBJPad is released?
Thanks

**Comment by josephsl** [comment]
Hi, let’s discuss ObjPad somewhere else please to avoid going off-topic. Thanks.

**Comment by cary-rowen** [comment]
then please hide your comment, i already did

**Comment by josephsl** [comment]
Hi,

For now, closing as complete via input gestures. Thanks.
